app/src/main/java/de/berlindroid/zepatch/MainActivity.kt

```diff
@@ -173,6 +175,10 @@ private fun PatchableDetail(
     onBackClick: () -> Unit,
     patchable: @Composable () -> Unit,
 ) {
+    var imageBitmap by remember { mutableStateOf<ImageBitmap?>(null) }
```

Comment:
This is the hoisted state so the sub steps remember what happened before and don't have to rework things.